test/mailers/developer_mailer_test.rb

```diff
@@ -2,14 +2,19 @@
 
 class DeveloperMailerTest < ActionMailer::TestCase
   test "stale" do
-    Developers::ProfileReminderNotification.with(developer:).deliver(recipient)
+    assert_emails 1 do
+      Developers::ProfileReminderNotification.with(developer:).deliver(recipient)
+    end
   end
 
   test "welcome" do
-    Developers::WelcomeNotification.with(developer:).deliver(recipient)
+    assert_emails 1 do
+      Developers::WelcomeNotification.with(developer:).deliver(recipient)
+    end
   end
 
   test "first message" do
+    # TODO: Make an assertion here
```

Comment:
Let's remove this TODO and create a new issue in the repo.

Reply:
Created as a bug